src/sdk/namespace_nb.js

```diff
@@ -312,6 +312,18 @@ class NamespaceNB {
     put_object_retention(params, object_sdk) {
         return object_sdk.rpc_client.object.put_object_retention(params);
     }
+
+    ////////////
+    //  ACLs  //
+    ////////////
+
+    get_object_acl(params, object_sdk) {
+        return object_sdk.rpc_client.object.get_object_acl(params);
```

Comment:
You are referring to API's that do not exist on object_api...

Reply:
Should they be added to object_api then?

Reply:
```
const res = await req.object_sdk.read_object_md({ .. });
return { 
  owner: s3_utils.DEFAULT_S3_USER, 
  access_control_list: [{ 
    Grantee: { ID: s3_utils.DEFAULT_S3_USER, Type: "CanonicalUser" }, 
    Permission: "FULL_CONTROL" 
  }]
};
```